glide.yaml

```diff
@@ -12,3 +12,7 @@ import:
   version: master
 - package: github.com/giantswarm/operatorkit
   version: master
+- package: github.com/prometheus/prometheus
+  version: v1.6.3
```

Comment:
Went with our current Prometheus version. I reckon we should track here against our deployed version, to match up the configuration versions.